rubocop.el

```diff
@@ -140,9 +140,17 @@ Alternatively prompt user for directory."
   (interactive)
   (rubocop--file-command rubocop-autocorrect-command))
 
+(defun rubocop-bundled-p ()
+  "Check if rubocop has been bundled."
+  (let ((gemfile-lock (expand-file-name "Gemfile.lock" (rubocop-project-root))))
+    (when (file-exists-p gemfile-lock)
+      (with-temp-buffer
+        (insert-file-contents gemfile-lock)
+        (re-search-forward "rubocop" nil t)))))
+
 (defun rubocop-ensure-installed ()
   "Check if RuboCop is installed."
-  (unless (executable-find "rubocop")
+  (unless (or (executable-find "rubocop") (rubocop-bundled-p))
```

Comment:
@bbatsov I decided to work around that by creating a function that `rubocop--file-command` and `rubocop--dir-command` delegate to. Seems to work well once I set my `rubocop-check-command` back to the default value. Let me know if I can do anything else. Thanks!